src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -178,36 +178,60 @@ public void SetupFunctionInvocationBuffers(IEnumerable<FunctionMetadata> functio
             }
         }
 
-        public void SendFunctionLoadRequests()
+        public async Task SendFunctionLoadRequests()
         {
-            if (_functions != null)
+            var initialized = await _workerInitTask.Task;
```

Comment:
Prevents function load (which also blocks execution) from happening before worker is ready